cassandra/src/test/scala/org/apache/zeppelin/cassandra/ParagraphParserTest.scala

```diff
@@ -496,73 +545,180 @@ class ParagraphParserTest extends FlatSpec
+  }
+
+  "Parser" should "fail parsing describe function" in {
+    val queries ="Describe function toto"
 
     val ex = intercept[InterpreterException] {
       parser.parseAll(parser.queries, queries)
     }
-    ex.getMessage should be(s"Invalid syntax for DESCRIBE TABLE. It should comply to the patterns: " +
-      s"${DESCRIBE_TABLE_WITH_KEYSPACE_PATTERN.toString} or ${DESCRIBE_TABLE_PATTERN.toString}")
+    ex.getMessage should be(s"Invalid syntax for DESCRIBE FUNCTION. It should comply to the patterns: " +
+      s"${DESCRIBE_FUNCTION_WITH_KEYSPACE_PATTERN.toString} or ${DESCRIBE_FUNCTION_PATTERN.toString}")
   }
 
-  "Parser" should "parse describe type" in {
-    val queries ="Describe Type toto;"
+  "Parser" should "parse describe functions" in {
+    val queries ="Describe functions;"
 
     val parsed = parser.parseAll(parser.queries, queries)
 
-    parsed.get should be(List(DescribeUDTCmd(None,"toto")))
+    parsed.get(0) shouldBe a [DescribeFunctionsCmd]
   }
 
-  "Parser" should "parse describe type with keyspace" in {
-    val queries ="Describe Type ks.toto;"
+  "Parser" should "fail parsing describe functions" in {
+    val queries ="Describe functions toto"
+
+    val ex = intercept[InterpreterException] {
+      parser.parseAll(parser.queries, queries)
+    }
+    ex.getMessage should be(s"Invalid syntax for DESCRIBE FUNCTIONS. It should comply to the pattern: " +
+      s"${DESCRIBE_FUNCTIONS_PATTERN.toString}")
+  }
+
+
+  "Parser" should "parse describe aggregate" in {
+    val queries ="Describe aggregate toto;"
 
     val parsed = parser.parseAll(parser.queries, queries)
 
-    parsed.get should be(List(DescribeUDTCmd(Some("ks"),"toto")))
+    parsed.get should be(List(DescribeAggregateCmd(None,"toto")))
   }
 
-  "Parser" should "fail parsing describe type" in {
-    val queries ="Describe Type toto"
+  "Parser" should "parse describe aggregate with keyspace" in {
+    val queries ="Describe aggregate ks.toto;"
+
+    val parsed = parser.parseAll(parser.queries, queries)
+
+    parsed.get should be(List(DescribeAggregateCmd(Some("ks"),"toto")))
+  }
+
+  "Parser" should "fail parsing describe aggregate" in {
+    val queries ="Describe aggregate toto"
 
     val ex = intercept[InterpreterException] {
       parser.parseAll(parser.queries, queries)
     }
-    ex.getMessage should be(s"Invalid syntax for DESCRIBE TYPE. It should comply to the patterns: " +
-      s"${DESCRIBE_TYPE_WITH_KEYSPACE_PATTERN.toString} or ${DESCRIBE_TYPE_PATTERN.toString}")
+    ex.getMessage should be(s"Invalid syntax for DESCRIBE AGGREGATE. It should comply to the patterns: " +
+      s"${DESCRIBE_AGGREGATE_WITH_KEYSPACE_PATTERN.toString} or ${DESCRIBE_AGGREGATE_PATTERN.toString}")
+  }
+
+  "Parser" should "parse describe aggregates" in {
+    val queries ="Describe aggregates;"
+
+    val parsed = parser.parseAll(parser.queries, queries)
+
+    parsed.get(0) shouldBe a [DescribeAggregatesCmd]
```

Comment:
Same remark for ** DescribeAggregatesCmd**, no property object. Ok with the .get(0)